pages/your_tasks/data_workflow_diagram.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Data workflow diagram
+contributors: [Vilem Ded]
+description: Best practices to capture your planned data workflow in a diagram.
+page_id: data_workflow_diagram
+related_pages: 
+dsw:
+faircookbook:
+---
+
+## Why you should draw a data workflow diagram?
```

Comment:
I don't think all your solution fall under why we should have one. Shoul we not also have a section about how?

Reply:
I am not a native speaker, but "Why should you draw a data workflow diagram?" sounds better to my ears, but this can be on me!

Reply:
right! How makes sense. Also, it aligns well with the structure of the other task pages.